BIDebugEngine/BIDebugEngine/NetworkController.cpp

```diff
@@ -217,30 +228,24 @@ void NetworkController::incomingMessage(const std::string& message) {
             } break;
 
             case NC_CommandType::LoadFile: {
-                if (GlobalDebugger.state != DebuggerState::breakState) {
-                    answer.Serialize("exception", "getCurrentCode: Not in breakState!");
-                    return;
-                }
-
-
-                intercept::client::invoker_lock lock(true);
-                if (GlobalDebugger.state != DebuggerState::breakState) lock.lock();
-
-                JsonArchive ar(packet["data"]);
-
-                r_string path;
-                ar.Serialize("path", path);
-
-                auto result = intercept::sqf::load_file(path);
-                JsonArchive answerAr;
-                answerAr.Serialize("path", path);
-                answerAr.Serialize("content", result);
-
                 JsonArchive answer;
                 answer.Serialize("handle", packet.value<std::string>("handle", {}));
-                answer.Serialize("data", answerAr);
                 answer.Serialize("command", static_cast<int>(NC_OutgoingCommandType::LoadFileResult));
-
+                if (GlobalDebugger.state != DebuggerState::breakState) {
+                    answer.Serialize("exception", "getCurrentCode: Not in breakState!");
+                } else {
+                    JsonArchive ar(packet["data"]);
+                    intercept::client::invoker_lock lock(true);
+                    if (GlobalDebugger.state != DebuggerState::breakState) lock.lock();
```

Comment:
that check and the lock are useless now

Reply:
Removed in 63936536b8b5079544c82aaec932ad66cca90577.